base/src/com/google/idea/blaze/base/projectview/ProjectViewSet.java

```diff
@@ -52,10 +52,12 @@ public <T> List<T> listItems(SectionKey<T, ListSection<T>> key) {
   }
 
   /** Returns all values from all scalar sections in the project views, in order */
-  public <T> List<T> listScalarItems(SectionKey<T, ScalarSection<T>> key) {
+  public <T> List<T> listScalarItems(SectionKey<T, ScalarSection<T>>... keys) {
```

Comment:
not sure whether this is a good idea so can redo if someone have strong opinions against this way